### PR TITLE
offchain - place ccip libraries under internal package

### PR DESCRIPTION
**Pull request by dimkouv**

### CODE REVIEW

**Review by mateusz-sekara — COMMENTED** [review]
What's the reasoning behind moving all those pkgs into a single `internal` directory?

File: .gitignore [hunk 1]
```diff
@@ -75,7 +75,7 @@ MacOSX*
 cache
```
Comment (dimkouv) [inline .gitignore:75]: @mateusz-sekara > What's the reasoning behind moving all those pkgs into a single internal directory?

cleaning up the repo, step-by-step, followup prs will come, moving under internal/ makes this libs non-importable by other non-ccip packages

Comment (dimkouv) [reply .gitignore:75]: it's going to be something like this
```
commit_plugin/
exec_plugin/
internal/
   libs/
ccip.go
```

**Review by mateusz-sekara — APPROVED** [review]

**Review by connorwstein — APPROVED** [review]

**Comment by dimkouv** [comment]
moved to https://github.com/smartcontractkit/ccip/pull/133 due to some issue with submodules

**Pull request closed without merging**